cmd/preflight/cmd/check_container_test.go

```diff
@@ -7,49 +7,49 @@ import (
 	"os"
 	"path/filepath"
 
-	"github.com/redhat-openshift-ecosystem/openshift-preflight/artifacts"
 	"github.com/redhat-openshift-ecosystem/openshift-preflight/certification"
 	"github.com/redhat-openshift-ecosystem/openshift-preflight/internal/check"
 	"github.com/redhat-openshift-ecosystem/openshift-preflight/internal/cli"
 	"github.com/redhat-openshift-ecosystem/openshift-preflight/internal/formatters"
 	"github.com/redhat-openshift-ecosystem/openshift-preflight/internal/lib"
-	"github.com/redhat-openshift-ecosystem/openshift-preflight/internal/viper"
 
 	"github.com/go-logr/logr"
 	. "github.com/onsi/ginkgo/v2"
 	. "github.com/onsi/gomega"
+	"github.com/spf13/afero"
+	"github.com/spf13/viper"
 )
 
 var _ = Describe("Check Container Command", func() {
 	BeforeEach(createAndCleanupDirForArtifactsAndLogs)
 
-	Context("when running the check container subcommand", func() {
-		Context("With all of the required parameters", func() {
+	When("running the check container subcommand", func() {
```

Comment:
WOAH, Woah, woah... unrelated change alert?! :laughing:

Reply:
Is it though? :)